prime-router/settings/staging/0166-flexion-staging-results-handling.yml

```diff
@@ -146,3 +149,30 @@
         authHeaders:
           Content-Type: "application/x-www-form-urlencoded"
           Host: "cdcti-stg-api.azurewebsites.net"
+    # RS => Hospital: translates ORU_R01 from FHIR to HL7 and uploads the message to the SFTP folder
+    - name: "simulated-hospital"
+      externalName: "Simulated Hospital ETOR Receiver - Receiving ORU_R01 HL7 results"
+      organizationName: "flexion"
+      topic: "etor-ti"
+      customerStatus: "active"
+      jurisdictionalFilter:
+        - "Bundle.entry.resource.ofType(MessageHeader).event.code = 'R01'" # ORU_R01
+        - "Bundle.entry.resource.ofType(MessageHeader).meta.tag.where(system = 'http://localcodes.org/ETOR').code = 'ETOR'"
+        # - "Bundle.entry.resource.ofType(MessageHeader).destination.receiver.resolve().identifier.where(value = 'simulated-hospital').exists()"
```

Comment:
Do we know which receiver we're filtering for? Does this need to stay commented until we know?

Reply:
Hmmm I was planning to leave it commented just as an example that we can follow for actual partner settings, but it may be a good idea to use it. `simulated-hospital` and `simulated-hospital` will only be used by us for testing purposes, so we can decide on an id that we'll use in the `MSH-6.2` for our testing messages

Reply:
I uncommented the filters. When sending test messages to staging we'll now need to use `simulated-hospital-id` in `MSH-6.2` when sending results and `simulated-lab-id` in `MSH-6.2` when sending orders